src/encoded/static/components/item-pages/MicroscopeConfigurationView.js

```diff
@@ -574,6 +591,457 @@ export class MicroMetaTabView extends React.PureComponent {
+
+                tableHeader = (
+                    <div className="row summary-sub-header">
+                        <div className="col col-lg-3 col-xl-3 summary-title-column text-truncate">MetaData {showPrevBtn ? this.prevNextButton(false) : null}</div>
+                        {_.map(visibleMatches, function (m, index) {
+                            return (
+                                <div className={columClassName + " summary-title-column text-truncate"}>
+                                    {m.Name}
+                                </div>
+                            );
+                        }, this)}
+                        {showNextBtn ? <div className="col-lg-1 col-xl-1 summary-title-column text-truncate">{this.prevNextButton(true)}</div> : null}
+                    </div>
+                );
+            }
+            
+            const schemaPropPairs = _.pairs(schema.properties);          
+
+            tableBody = _.map(_.keys(schema.subCategoriesOrder), function(subCategory){
+                const subCategoryProperties = _.filter(schemaPropPairs, function (spp) {
+                    const [, propItem] = spp;
+                    return propItem && propItem.category === subCategory;
+                });
+                const sectionProps = {
+                    subCategory, subCategoryProperties, matches: visibleMatches, tooltipLimit, columClassName, 
+                    collapsed: !!collapsedSections[subCategory]
+                };
+                return <CollapsibleSubCategory key={subCategory} {...sectionProps} toggleExpand={this.toggleExpand} />
+            }, this);
+
+            // update title to include selected term name and count
+            if (!isMobileSize && currentFilters && currentFilters.length > 0) {
+                const [{ term: filterTerm }] = currentFilters;
+                headerTitle += ` - ${filterTerm} (${matches.length})`;
+            }
+        } else {
+            placeholderStyle.width = null;
+            placeholderStyle.height = null;
+            if (typeof height === 'number' && height >= 400) {
+                placeholderStyle.height = height - 200;
+                placeholderStyle.paddingTop = ((height - 200) / 2) - 40;
+            }
+            tableBody = (
+                <div className="text-center" style={placeholderStyle}>
+                    <MicroMetaLoadingIndicator title="Select a Component to Activate the Summary Table" />
+                </div>);
+        }
+
+        const facetListProps = {
+            context: { "filters": currentFilters },
+            title: "Hardware Explorer",
+            facets,
+            onFilter: this.onFilter,
+            useRadioIcon: true,
+            persistSelectedTerms: false
+        };
+
+        return (
+            <div class="overflow-hidden">
+                <h3 class="tab-section-title"><span>Hardware Summary</span></h3>
```

Comment:
Need to use `className`, not `class`.

(Do you guys have ESLint or any other linting setup on VSCode? It can help catch small stuff like this).

Reply:
(Updated to 'className' / pushed)

Reply:
@alexkb0009 My bad, it looks that my ESLint is failing for a while since I moved my working folder and VS Code is not matching with folder name in .vscodee/setting.json. Thank you.